utils/emacs/carbon-mode.el

```diff
@@ -0,0 +1,127 @@
+;;; carbon-mode.el --- major mode for editing Carbon source code  -*- lexical-binding: t; -*-
+
+;; Version: 0.0.0
+;; Author: Lesley Lai
+;; Url: https://github.com/rust-lang/rust-mode
```

Comment:
Should fix this incorrect reference to rust-mode

Reply:
Oops! Thanks!

Now you know which mode that I "consulted" 😅.